frontend/src/components/activity/content/SafetyConcept.vue

```diff
@@ -7,7 +7,7 @@
       rows="2"
       :disabled="layoutMode || disabled"
       :filled="layoutMode"
-      height="100%"
+      class="grow-v-slot"
```

Comment:
Sometimes you use `class=`, sometimes you use `input-class=`.
Seems both do the job, but might be better to be consistent. No?

Reply:
Thanks good point.